### PR TITLE
feat(series-search): add focus/blur menu to search

### PR DESCRIPTION
**Pull request by davidholyko**
#### Summary

- Work Type: Feature 

#### What I did

- User can dismiss series results menu now by clicking away
- Fixed minor visual bugs

#### Notes / Research / References

- Detecting click outside: https://stackoverflow.com/questions/54391682/detect-click-outside-component-react-hooks
-

#### Checklist

- [x] Branch is in format `TYPE/scope/description`
- [x] PR title is in format `TYPE(scope): description`
- [x] Commits are in semantic format
- [x] Has Summary

### CODE REVIEW

File: src/scss/mixins.scss [hunk 1]
```diff
@@ -16,15 +16,15 @@
 
   &:disabled {
     background-color: $white;
-    border-color: $dark-grey;
+    border-color: $light-grey;
```
Comment (davidholyko) [inline src/scss/mixins.scss:19]: light grey fixes visual accessibility

File: src/scss/SeriesSearch.scss [hunk 2]
```diff
@@ -17,6 +17,7 @@
   border-radius: 3px;
   border: none;
   width: 50vw;
+  z-index: 50;
```
Comment (davidholyko) [inline src/scss/SeriesSearch.scss:20]: fixes buttons from appear over series search results dropdown menu

File: src/components/TvShowPoster/TvShowPosterSection.js [hunk 3]
```diff
@@ -1,22 +1,20 @@
...
 import { TvShowPosterSkeleton } from './TvShowPosterSkeleton';
 
 export const TvShowPosterSection = ({ data, heading }) => {
-  const { isLoading } = useContext(AppContext);
   return (
     <div>
       <h1>{heading}</h1>
       <div className="tv-show-poster-section">
-        {isLoading ? (
+        {data && !data.length ? (
```
Comment (davidholyko) [inline src/components/TvShowPoster/TvShowPosterSection.js:17]: fixes bug where skeletons would always show on loading, reverted work from https://github.com/starter-code/binge/pull/53

File: src/scss/bootstrap.scss [hunk 4]
```diff
@@ -0,0 +1,7 @@
+.visible {
```
Comment (davidholyko) [inline src/scss/bootstrap.scss:1]: changed name from `Bootstrap` to `bootstrap`. Bootstrap is not a component

File: src/components/SeriesSearch/SeriesSearchForm.js [hunk 5]
```diff
@@ -1,14 +1,37 @@
...
+    return () => {
+      document.removeEventListener('click', handleClickOutside, false);
+    };
+  }, []);
+
+  const handleClickOutside = (event) => {
+    const resultsRef = searchResultsRef.current;
+    const inputRef = searchInputRef.current;
+
+    if (
```
Comment (davidholyko) [inline src/components/SeriesSearch/SeriesSearchForm.js:27]: If the user clicks outside the two provided refs, close the menu

**Review by simonchen92 — APPROVED** [review]
LGTM

**Pull request merged**